idunn/places/pj_poi.py

```diff
@@ -273,28 +273,47 @@ def get_phone(self):
             None,
         )
 
-    def get_website(self):
-        if not self.data.website_urls:
-            return None
+    def get_website_url_for_type(self, social_network: UrlType) -> Optional[str]:
+        return next(
+            (
+                resolve_url(website.website_url)
+                for website in self.data.website_urls or []
+                if website.url_type == social_network
+            ),
+            None,
+        )
 
-        return resolve_url(self.data.website_urls[0].website_url)
+    def get_website(self):
+        return self.get_website_url_for_type(UrlType.EXTERNAL_WEBSITE)
 
     def get_website_label(self):
-        if not self.data.website_urls:
-            return None
-
         if isinstance(self.data, pj_find.Listing):
             # FIXME: Ideally the Listing would include a "suggested_label" too
             return self.get_local_name()
 
-        suggested_label = self.data.website_urls[0].suggested_label
+        suggested_label = self.get_website_url_for_type(UrlType.EXTERNAL_WEBSITE)
```

Comment:
So the `suggested_label` field provided by the API is no longer used ?

Reply:
That's fixed, and also the lack of test :sweat_smile: